wire-ios/Wire-iOS/Sources/Authentication/Interface/Helpers/TextFieldValidator.swift

```diff
@@ -51,7 +57,7 @@ final class TextFieldValidator {
         }
     }
 
-    func validate(text: String?, kind: ValidatedTextField.Kind) -> TextFieldValidator.ValidationError? {
+    func validate(text: String?, kind: ValidatedTextField.Kind, useGuestLinkRuleset: Bool = false) -> TextFieldValidator.ValidationError? {
```

Comment:
I would create an enum with a ruleset type instead of a property. It will be more expandable in the future.